src/background.js

```diff
@@ -27,3 +27,31 @@ browser.runtime.onMessage.addListener(async message => {
 
 // GitHub Enterprise support
 injectContentScripts();
+
+browser.contextMenus.create({
+	id: 'enable-extension-on-new-domain',
+	title: 'Enable Refined GitHub on this domain',
+	contexts: ['page_action'],
+	documentUrlPatterns: [
+		'http://*/*',
+		'https://*/*'
+	]
+});
+
+browser.contextMenus.onClicked.addListener(async ({menuItemId}, {tabId, url}) => {
+	/* eslint-disable no-alert */
+	/* global chrome */
+	if (menuItemId === 'enable-extension-on-new-domain') {
+		chrome.permissions.request({
+			origins: [
+				`${new URL(url).origin}/*`
+			]
+		}, granted => {
+			if (chrome.runtime.lastError) {
+				alert(`Error: ${chrome.runtime.lastError}`);
+			} else if (granted && confirm('Reload this page to apply Refined GitHub?')) {
```

Comment:
I don't think `?` is correct here.

Reply:
No? 

<img width="461" alt="screen shot 2017-11-16 at 13 08 46" src="https://user-images.githubusercontent.com/1402241/32874781-4e378544-cacf-11e7-8f94-00beb756fe93.png">

Reply:
Ok, I misunderstood. I think something like this would be clearer:

> Do you want to reload this page to apply Refined GitHub?

Reply:
58f0602 ✅